tests/integration/bigquery.spec.ts

```diff
@@ -15,7 +15,10 @@ suite("@dataform/integration/bigquery", { parallel: true }, ({ before, after })
   let dbadapter: BigQueryDbAdapter;
 
   before("create adapter", async () => {
-    dbadapter = (await dbadapters.create(credentials, "bigquery")) as BigQueryDbAdapter;
+    dbadapter = (await dbadapters.create(
+      { ...credentials, location: "EU" },
```

Comment:
why do you need to do this if it's in the credentials file already?

Reply:
Credentials uploaded to the web UI don't have location specified, so I've taken them out of the encoded credentials file too to make the tests closer to our actual use case